src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
Sorry I don't understand the problem with the model. In daydream you can only have one controller and the app cannot make any assumption about the hand since it's a user preference. The model you use should be hand agnostic. I would remove this property from the schema and reintroduce it later if there's a legitimate use case.